ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-cloud_hotfix_releases-presubmits.yaml

```diff
@@ -88,6 +88,7 @@ presubmits:
       job-release: "4.18"
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-assisted-service-cloud_hotfix_releases-e2e-metal-assisted-4-13
+    optional: true
```

Comment:
There is an annoying bug `ci-operator` that when you have a job already and you change its `always_run` field, it doesn't reflect in the actual config, see above line 77. To overcome this you should remove the entire job, run `make update` then add the job back with `always_run: false` then run `make update` again